packages/@lwc/shared/src/language.ts

```diff
@@ -97,6 +98,7 @@ export {
     create,
     defineProperties,
     defineProperty,
+    entries,
```

Comment:
`Object.entries` is supported in all [our supported browsers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries); we had just never used it before.